### PR TITLE
[YUNIKORN-2490] update PMC and committers

### PR DESCRIPTION
**Pull request by wilfred-s**
### What is this PR for?
Adding new committers and PMC member

### What type of PR is it?
* [X] - Documentation

### What is the Jira issue?
* https://issues.apache.org/jira/browse/YUNIKORN-2490

### How should this be tested?
NA

### CODE REVIEW

**Comment by wilfred-s** [comment]
@brandboat: you did not show up in the reviewer list yet. Please check if I set the correct details.

**Review by chia7712 — COMMENTED** [review]
My personal info is correct. thanks @wilfred-s

File: src/pages/community/people.md [hunk 1]
```diff
@@ -66,14 +67,16 @@ members and [committers](https://infra.apache.org/new-committers-guide.html#what
...
-| Ted Lin        | steinsgateted | NCKU         | +8                  |
-| Qi Zhu         | zhuqi         | Cloudera     | +8                  |
+| Name          | ApacheID      | Organization | Timezone <br/>(UTC) |
+|---------------|---------------|--------------|---------------------|
+| Hsuan Zong Wu | samhxwu       |              | +8                  |
+| Kuan-Po Tseng | brandboat     | SUSE         | +8                  |
+| Li Gao        | ligao         | Databricks   | -8                  |
+| PoAn Yang     | payang        |              | +8                  |
+| Qi Zhu        | zhuqi         | Cloudera     | +8                  |
+| Ryan Lo       | wenchih       | NCKU         | +8                  |
```
Comment (lowc1012) [inline src/pages/community/people.md:77]: Hi @wilfred-s
My org is `Fazz` and everything else is correct.
Thank you!

**Comment by brandboat** [comment]
> @brandboat: you did not show up in the reviewer list yet. Please check if I set the correct details.

Thank you for the reminder. I've just finished setting up the related GitHub configurations.
And my personal detail is correct. Thank you @wilfred-s

**Pull request closed without merging**